docker-app/qfieldcloud/core/paginators.py

```diff
@@ -35,3 +36,7 @@ def count(self):
             else:
                 return estimate
         return len(self.object_list)
+
+
+class DefaultPagination(pagination.LimitOffsetPagination):
+    limit_query_param = "items_per_page"
```

Comment:
```suggestion
class LimitOffsetPagination(pagination.LimitOffsetPagination):
    ...
```
Just checked the base class has `limit_query_param="limit"`, which is ok name, I think there is no need to replace it. The one on ClickUp was just an example name.